lib/hsa/hsaBeamformKernel.cpp

```diff
@@ -250,6 +253,11 @@ hsa_signal_t hsaBeamformKernel::execute(int gpu_frame_id, hsa_signal_t precede_s
     params.private_segment_size = 0;
     params.group_segment_size = 16384;
 
+    //KV
+    params.workgroup_size_x = 64;
+    params.grid_size_x = 64;
+    params.group_segment_size = 2*512*sizeof(float) + 16 * 2 * sizeof(float);
```

Comment:
This appears to just overwrite the values above, we should only have one copy of this.

Reply:
Yup, it was split up for easier flipping back and forth, but it should just be one copy, totally agreed.